package/MDAnalysis/lib/distances.py

```diff
@@ -24,19 +24,60 @@
 Functions
 ---------
 
-.. autofunction:: distance_array(reference, configuation [, box [,result]])
-.. autofunction:: self_distance_array(reference [, box [,result]])
-.. autofunction:: calc_bonds(atom1, atom2 [, box, [,result]])
-.. autofunction:: calc_angles(atom1, atom2, atom3 [,box [, result]])
-.. autofunction:: calc_dihedrals(atom1, atom2, atom3, atom4 [,box [, result]])
-.. autofunction:: apply_PBC(coordinates, box)
-.. autofunction:: transform_RtoS(coordinates, box)
-.. autofunction:: transform_StoR(coordinates, box)
+All functions take the optional keyword *backend*, which determines the
+type of acceleration. Currently, the following choices are implemented:
+
+========== ======================== ======================================
+*backend*  module                   description
+========== ======================== ======================================
+"serial"   :mod:`_distances`        serial implementation in C/Cython
+
+"OpenMP"   :mod:`_distances_openmp` parallel implementation in C/Cython
+                                    with OpenMP
+========== ======================== ======================================
+
+.. autofunction:: distance_array(reference, configuration [, box [, result [, backend]]])
+.. autofunction:: self_distance_array(reference [, box [,result [, backend]]])
+.. autofunction:: calc_bonds(atom1, atom2 [, box, [, result [, backend]]])
+.. autofunction:: calc_angles(atom1, atom2, atom3 [,box [, result [, backend]]])
+.. autofunction:: calc_dihedrals(atom1, atom2, atom3, atom4 [,box [, result [, backend]]])
+.. autofunction:: apply_PBC(coordinates, box [, backend])
+.. autofunction:: transform_RtoS(coordinates, box [, backend])
+.. autofunction:: transform_StoR(coordinates, box [,backend])
 """
 import numpy as np
 from numpy.lib.utils import deprecate
 
 from .mdamath import triclinic_vectors, triclinic_box
+
+
+# hack to select backend with backend=<backend> kwarg. Note that
+# the cython parallel code (prange) in parallel.distances is
+# independent from the OpenMP code
+import importlib
+_distances = {}
+_distances['serial'] = importlib.import_module("._distances",
+                                         package="MDAnalysis.lib")
+try:
+    _distances['OpenMP'] = importlib.import_module("._distances_openmp",
```

Comment:
This is pretty minor, but can we make the backend selection case insensitive?  I think you could just call this 'openmp' and then call `.lower()` on whatever string is passed.